src/objectTranslations/Contact-es.objectTranslation

```diff
@@ -1,129 +1,138 @@
+        <help>El importe del primer crédito blando de este contacto.</help>
+        <label>Importe del primer crédito blando</label>
         <name>First_Soft_Credit_Amount__c</name>
     </fields>
     <fields>
-        <help>Fecha del primer crédito blando</help>
+        <help>La fecha del primer crédito blando de este contacto.</help>
         <label>Fecha del primer crédito blando</label>
         <name>First_Soft_Credit_Date__c</name>
     </fields>
     <fields>
-        <help>Contiene el ID de la familia, ya sea que se usen cuentas o el objeto familias.</help>
-        <label>Id de familia</label>
+        <help>Contiene el Id. del registro del hogar (ya sea la cuenta del hogar o el objeto Hogar) con el que este contacto está asociado en estos momentos (solo lectura).</help>
+        <label>Id. de Hogar</label>
         <name>HHId__c</name>
     </fields>
     <fields>
-        <help>El monto del mayor crédito blando</help>
-        <label>Mayor monto de crédito blando</label>
+        <help>El importe más gran del crédito blando de este contacto.</help>
+        <label>Importe más grande de crédito blando</label>
         <name>Largest_Soft_Credit_Amount__c</name>
     </fields>
     <fields>
-        <help>Fecha del mayor crédito blando</help>
-        <label>Fecha del mayor crédito blando</label>
+        <help>La fecha del crédito blando más grande de este contacto.</help>
+        <label>Fecha del crédito blando más grande</label>
         <name>Largest_Soft_Credit_Date__c</name>
     </fields>
     <fields>
-        <help>Monto el último crédito blando</help>
-        <label>Monto el último crédito blando</label>
+        <help>El importe del último crédito blando de este contacto.</help>
+        <label>Importe del último crédito blando</label>
         <name>Last_Soft_Credit_Amount__c</name>
     </fields>
     <fields>
-        <help>Fecha del último crédito blando</help>
+        <help>La fecha del último crédito blando de este contacto.</help>
         <label>Fecha del último crédito blando</label>
         <name>Last_Soft_Credit_Date__c</name>
     </fields>
     <fields>
-        <help>La cantidad de créditos blandos, cerradas en los últimos días definidos en la configuración del NPSP</help>
-        <label>Cantidad de créd. blandos últimos N días</label>
+        <help>El número de créditos blandos atribuidos a este contacto que se cerraron en los últimos N días. El valor de N se define en Configuración de NPSP | Donaciones | Datos estadísticos de donantes.</help>
+        <label>Número créd. blandos en últimos N días</label>
         <name>Number_of_Soft_Credits_Last_N_Days__c</name>
     </fields>
     <fields>
-        <help>Cantidad de créditos blandos el año pasado</help>
-        <label>Cantidad de créd. blandos año pasado</label>
+        <help>El número de créditos blandos atribuidos a este contacto en el año de calendario anterior.</help>
+        <label>Número de créditos blandos el año pasado</label>
```

Comment:
@force2b Do you happen to know what this label looks like in the UI? I wonder if the previous translation used the abbreviation intentionally for creditos bcuz there's some limited space in the UI.

Reply:
Just so I'm sure, are you referring to the label or the help text on the `Number_of_Soft_Credits_Last_Year__c` field?

Reply:
"label" - it was previously abbreviated to "cred." instead of "creditos (new translation)"

Reply:
@GloryChoe How is this screen capture:
![image](https://user-images.githubusercontent.com/801017/70668145-bb0e9080-1c40-11ea-8a77-262eb9d3438c.png)